astacus/coordinator/api.py

```diff
@@ -30,6 +30,7 @@ class OpName(StrEnum):
     """(Long-running) operations defined in this API (for coordinator)"""
 
     backup = "backup"
+    delta_backup = "deltabackup"
```

Comment:
Can this be kept consistent with the name and use `delta_backup` for the value or do we need the non-underscore ?